hack/make/binary-daemon

```diff
@@ -14,7 +14,7 @@ copy_binaries() {
 		return
 	fi
 	echo "Copying nested executables into $dir"
-	for file in containerd containerd-shim ctr runc docker-init docker-proxy rootlesskit rootlesskit-docker-proxy dockerd-rootless.sh; do
+	for file in containerd containerd-shim containerd-shim-runc-v2 ctr runc docker-init docker-proxy rootlesskit rootlesskit-docker-proxy dockerd-rootless.sh; do
```

Comment:
Looks like you accidentally pulled in another change; probably if your branch was out of date; a rebase should fix that

Reply:
Indeed :-(
Fixed. Thank you.